### PR TITLE
Fix environ vars

### PR DESCRIPTION
**Pull request by kpy21ui**

### CODE REVIEW

File: sitemap_generate/generator.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def record(self, url: str) -> bytes:
         environ = {
             'REQUEST_METHOD': 'GET',
             'wsgi.input': '',
+            'wsgi.url_scheme': defaults.SITEMAP_PROTO,
```
Comment (tumb1er) [inline sitemap_generate/generator.py:53]: According to [PEP-333](https://www.python.org/dev/peps/pep-0333/#environ-variables) also missing required variables:
* [ ] `SCRIPT_NAME` - may be empty string
* [ ] `SERVER_PROTOCOL` - `"HTTP/1.1"`
* [ ] `wsgi.version` - `(1, 0)`
* [ ] `wsgi.errors` - `StringIO()` - same type as for `wsgi.input`
* [ ] `wsgi.multithread` - `False`
* [ ] `wsgi.multiprocess` - `False`
* [ ] `wsgi.run_once` - `False`

Could you please add these and fix `wsgi.input` type?

File: sitemap_generate/defaults.py [hunk 2]
```diff
@@ -3,6 +3,6 @@
 # Hostname used in sitemap links
 SITEMAP_HOST = e('SITEMAP_HOST', 'localhost')
 # Port used in sitemap links
-SITEMAP_PORT = int(e('SITEMAP_PORT', 443))
+SITEMAP_PORT = str(e('SITEMAP_PORT', 443))
```
[the remark this answers is not in the document]
Comment (kpy21ui) [reply sitemap_generate/defaults.py:6]: but some developers could override this variable in the `settings.py` with a type `int`. For robust behaviour, I casting this variable to `str` type

[the remark this answers is not in the document]
Comment (tumb1er) [reply sitemap_generate/defaults.py:6]: environment variables are not related to settings, see `os.getenv`. It's return value is `Optional[str]`.

[the remark this answers is not in the document]
Comment (kpy21ui) [reply sitemap_generate/defaults.py]: fixed

**Pull request merged**